events/2024-04-01-example-event.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: event
+title: My Training Event Title
+draft: true  # will hide your event from the GTN events list, remove once you are ready to announce your event
+
+description: |
+    Short one or two line description of the event
+
+cover-image: assets/images/gat.png
+cover-image-alt: GTN Logo on a spiral galaxy background with text galaxy admin training
+
+
+organizers: # GTN contributors or funders, must be defined in CONTRIBUTORS.yaml
+- shiltemann
+- hexylena
+
+tags: [Topic 1, Topic 2, 5-day course]
+
+date_start: 2024-04-01
+date_end: 2024-04-02 # optional, if event is more than one day
+location: Example street 42, City, Country
+cost: free
+audience: This event is intended for PhD students interested in Genomics. A basic knowledge in R is useful but not required.
+contact_email: organizers@example.com
+async: false # if asynchronous, we will not display the time columns on the program
+
+registraton:
+- link:
+  deadline:
+
+feedback:
+- link:
+  deadline:
+
+
+program:
+  - section: "Monday: Introduction"
+    description: Short description of the program.
+    tutorials:
+      - name: galaxy-intro-short
+        topic: introduction
+        time_start: "09:00"
+        time_end: "10:00"
+      - name: data-manipulation-olympics
+        topic: introduction
+        time_start: "10:00"
+      - type: custom
+        name: Coffee Break
+        time_start: "10:30"
+      - name: data-manipulation-olympics
+        topic: introduction
+        time_start: "10:45"
+        time_end: "12:00"
+      - name: ansible-galaxy
+        topic: admin
+        time_start: "13:00"
+      - type: custom
+        name: Custom Session
+        time_start: "16:00"
+        time_end: "17:00"
+        description: |
+          Description of the custom session here, in markdown, can add
+          [links](https://example.com) if needed
+      - type: custom
+        name: Wrap-up & Drinks
+        time_start: "17:00"
```

Comment:
I suppose some folks will want am/pm. Can we think of anything functional we want to do with the time? (calculating anything like course length or something?) Otherwise maybe we just have `time: "10am - 12pm", just a single plain text field for them to enter whatever they want.